boards/nucleo-common/include/board_common.h

```diff
@@ -41,16 +41,18 @@ extern "C" {
  * @{
  */
 #ifdef CPU_MODEL_STM32F302R8
+#define LED0_GPIO           GPIOB
```

Comment:
what's the difference to [here](https://github.com/RIOT-OS/RIOT/blob/621823c29d969fd02277e41a75f75a5bd4437a07/boards/stm32f7discovery/include/board.h#L41) which uses `LED0_PORT` not `LED0_GPIO` as you proposed. My point is not `LEDx_GPIO` vs. `LEDx_PORT` but rather that is should be consistently named in RIOT.